src/qml/QFieldCloudPopup.qml

```diff
@@ -304,6 +304,15 @@ Popup {
           titleText: detailsText.startsWith('[QF/') ? qsTr('A server error has occured, please try again.') : qsTr('A network error has occured, please try again.')
           detailsText: ''
 
+          function humanizeError(errorString) {
+            if (errorString.includes("Network Error")) {
+              errorString = "Network Error";
+            } else if (errorString.includes("Quota Error")) {
+              errorString = "Quota Error";
```

Comment:
Let's actually serve an error string that will make it super clear:

```suggestion
              errorString = "Your account's available storage is full.";
```

We might also want to add a suggestion on how to fix the situation. 

@m-kuhn , @suricactus , thoughts?

Reply:
Yes,that's much better understandable.
Link to the faq? Or even better directly to the respective page on qfc.

Reply:
Btw, should this be translatable?

Reply:
@m-kuhn , link to documentation is a good idea.

@mohsenD98 , I think the best way to introduce a link UX to this is to add a url string property in QfCollapsibleMessage and add some decided UI (maybe a button sitting at the right end of the message box) to launch the url in the external browser.

Reply:
@mohsenD98 , documentationFromErrorString